sgl-router/src/routers/grpc/harmony/stages/preparation.rs

```diff
@@ -169,6 +174,38 @@ impl HarmonyPreparationStage {
         Ok(None)
     }
 
+    /// Generate Harmony structural tag for structured output (text field)
+    ///
+    /// Converts text.format to structural tag that constrains the final channel.
+    /// Returns None if text.format is not specified or is "text".
+    fn generate_structural_output_tag_for_harmony(
```

Comment:
The naming of this function is a bit confusing. We have `generate_structural_output_tag_for_harmony` and `generate_harmony_structural_tag`.

The names are near identical.